test/Interop/Cxx/stdlib/use-std-iterator.swift

```diff
@@ -0,0 +1,46 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
+//
+// REQUIRES: executable_test
+//
+// Enable this everywhere once we have a solution for modularizing libstdc++: rdar://87654514
+// REQUIRES: OS=macosx
+
+import StdlibUnittest
+import StdIterator
+import std.iterator
+import StdVector
+import std.vector
+
+var StdIteratorTestSuite = TestSuite("StdIterator")
+
+StdIteratorTestSuite.test("init") {
+    var vector = Vector()
+    var _1: CInt = 1
+    vector.push_back(&_1)
```

Comment:
Are you getting errors when trying to pass `1` directly here?

Reply:
Yep, still getting a error when i pass an int

Reply:
this is the error i am getting:
`error: cannot pass immutable value as inout argument: literals are not mutable
    vector.push_back(1)
`

Reply:
@zoecarver how can i use `swift-ide-test` to dump the contents of the module `std.iterator`?